core-kotlin-modules/core-kotlin-datastructures/src/test/kotlin/com/baeldung/binarydecimal/BinaryDecimalConversionTest.kt

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.binarydecimal
+
+import org.junit.jupiter.api.Test
+import kotlin.math.pow
+import kotlin.test.assertEquals
+
+class BinaryDecimalConversionTest {
+
+    @Test
+    fun `binary to decimal using toInt() method`(){
+        assertEquals(439, "110110111".toInt(2))
+        assertEquals(109, "1101101".toInt(2))
+        assertEquals(27, "11011".toInt(2))
+        assertEquals(122979, "11110000001100011".toInt(2))
+    }
+
+    @Test
+    fun `binary to decimal mathematical approach`(){
+
```

Comment:
Let's remove the blank lines that occur after an opening brace and before a closing brace in all the methods of this class.